docs/data/date-pickers/custom-field/PickerWithBrowserField.js

```diff
@@ -3,10 +3,13 @@ import * as React from 'react';
 import { useSlotProps } from '@mui/base/utils';
 import Box from '@mui/material/Box';
 import Stack from '@mui/material/Stack';
+import IconButton from '@mui/material/IconButton';
 import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { DateRange as DateRangeIcon } from '@mui/x-date-pickers/internals';
```

Comment:
I think we should avoid using imports form `internals` in the doc
It gives a bad example to users

I would be in favor of either:

1. Duplicating the icon code here
2. Exporting it from the root
3. Moving it to `@mui/x-date-pickers/icons` (and re-exporting it from the root for consistency)

Reply:
Good point, thanks for spotting it. 👍 
I'd lean towards 3rd option. 🤔

Reply:
Done. 😉